src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -411,69 +408,79 @@ protected MetricStatistics[] InitializeArray(int length)
+        public MetricStatistics MeanAbsoluteError { get; }
 
         /// <summary>
         /// Summary Statistics for L2
         /// </summary>
-        public MetricStatistics L2 { get; }
+        public MetricStatistics MeanSquaredError { get; }
 
         /// <summary>
         /// Summary statistics for the root mean square loss (or RMS).
         /// </summary>
-        public MetricStatistics Rms { get; }
+        public MetricStatistics RootMeanSquaredError { get; }
 
         /// <summary>
         /// Summary statistics for the user-supplied loss function.
         /// </summary>
-        public MetricStatistics LossFn { get; }
+        public MetricStatistics LossFunction { get; }
 
         /// <summary>
         /// Summary statistics for the R squared value.
         /// </summary>
         public MetricStatistics RSquared { get; }
 
-        public RegressionMetricsStatistics()
+        internal RegressionMetricsStatistics()
         {
-            L1 = new MetricStatistics();
-            L2 = new MetricStatistics();
-            Rms = new MetricStatistics();
-            LossFn = new MetricStatistics();
+            MeanAbsoluteError = new MetricStatistics();
+            MeanSquaredError = new MetricStatistics();
+            RootMeanSquaredError = new MetricStatistics();
+            LossFunction = new MetricStatistics();
             RSquared = new MetricStatistics();
         }
 
         /// <summary>
         /// Add a set of evaluation metrics to the set of observations.
         /// </summary>
         /// <param name="metrics">The observed regression evaluation metric</param>
-        public override void Add(RegressionMetrics metrics)
+        void IMetricsStatistics<RegressionMetrics>.Add(RegressionMetrics metrics)
         {
-            L1.Add(metrics.MeanAbsoluteError);
-            L2.Add(metrics.MeanSquaredError);
-            Rms.Add(metrics.RootMeanSquaredError);
-            LossFn.Add(metrics.LossFunction);
+            MeanAbsoluteError.Add(metrics.MeanAbsoluteError);
+            MeanSquaredError.Add(metrics.MeanSquaredError);
+            RootMeanSquaredError.Add(metrics.RootMeanSquaredError);
+            LossFunction.Add(metrics.LossFunction);
             RSquared.Add(metrics.RSquared);
         }
     }
 
     /// <summary>
-    /// The BinaryClassificationMetricsStatistics class is computes summary
-    /// statistics over multiple observations of binary classification evaluation metrics.
+    /// The <see cref="BinaryClassificationMetricsStatistics"/> class holds summary
+    /// statistics over multiple observations of <see cref="BinaryClassificationMetrics"/>.
     /// </summary>
-    public sealed class BinaryClassificationMetricsStatistics : MetricsStatisticsBase<BinaryClassificationMetrics>
+    public sealed class BinaryClassificationMetricsStatistics : IMetricsStatistics<BinaryClassificationMetrics>
     {
         /// <summary>
         /// Summary Statistics for AUC
```

Comment:
`AUC` => `AreaUnderRocCurve`?

Reply:
I think what I should probably do here in the *documentation* is do a direct link against the relevant property in the corresponding metrics class, using `<see` tags. Let me update it to do that...